tools/clang/test/SemaHLSL/hlsl/types/toolong-vectors.hlsl

```diff
@@ -0,0 +1,116 @@
+// RUN: %dxc -T lib_6_9 -DTYPE=float -DNUM=1025 -verify %s
+// RUN: %dxc -T ps_6_9  -DTYPE=float -DNUM=1025 -verify %s
+
+// A test to verify that declarations of longvecs are permitted in all the accepted places.
+// Only tests for acceptance, most codegen is ignored for now.
```

Comment:
When you say "most codegen is ignored for now" do you mean until long vecs are implemented? Do we intend to use this hlsl file to verify codegen at some point? Just wondering if this comment is going to become stale.